launch

```diff
@@ -25,15 +26,25 @@ OVPSIM_OVERRIDES = {
     "uartCBUS/portnum": PORT
 }
 
-GDB = ['cgdb', '-d', TRIPLET + 'gdb']
+DEBUGGERS = {
+    'gdb'   : '%(triplet)s-gdb %(kernel)s',
+    'cgdb'  : 'cgdb -d %(triplet)s-gdb %(kernel)s',
+    'ddd'   : 'ddd --debugger %(triplet)s-gdb %(kernel)s',
+    'gdbtui': '%(triplet)s-gdb -tui',
+    'emacs' : 'emacsclient -c -e \'(gdb "%(triplet)s-gdb -i=mi %(kernel)s")\''
+}
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description='Launch OVPsim with Malta board.')
     parser.add_argument('kernel', metavar='KERNEL', type=str,
                         help='Kernel file in ELF format.')
     parser.add_argument('-d', '--debug', action='store_true',
-                        help='Start simulation under gdb using cgdb.')
+                        help='Start simulation under gdb.')
+    parser.add_argument('-D', metavar='DEBUGGER', type=str, dest='debugger',
+                        choices=DEBUGGERS.keys(), default='cgdb',
+                        help='Debugger to use in -d mode. Available options: '
+                        '"cgdb" (default), "gdb", "gdbtui", "ddd", "emacs".')
```

Comment:
If only that line could be generated from `DEBUGGER.keys()` then the change would be perfect :)